orchagent/port.h

```diff
@@ -119,7 +119,8 @@ class Port
     std::vector<sai_object_id_t> m_queue_ids;
     std::vector<sai_object_id_t> m_priority_group_ids;
     sai_port_priority_flow_control_mode_t m_pfc_asym = SAI_PORT_PRIORITY_FLOW_CONTROL_MODE_COMBINED;
-    uint8_t   m_pfc_bitmask = 0;
+    uint8_t   m_pfc_bitmask_cfg = 0;    // bitmask from config
+    uint8_t   m_pfc_bitmask_status = 0; // bitmask status in asic
```

Comment:
a better way to differentiate is to have two variables named as

m_pfc_bitmask_usercfg;
m_pfc_bitmask_wdcfg;


technicially, they are both config, one is user config, one is pfcwd config